Gemfile

```diff
@@ -105,7 +105,7 @@ group :development, :test do
   gem "spring-commands-rspec"
   gem "teaspoon-jasmine"
   gem "test-unit", "~> 3.0"
-  gem "thin"
+  gem "thin", "~> 1.7.2"
```

Comment:
Ah maybe we don't. I got in the habit of doing that because I thought it helped keep things in sync across forks--when do and don't we want to lock gemfile versions?